packages/select/docs/Select/Select.stories.tsx

```diff
@@ -61,6 +62,15 @@ DefaultStory.parameters = {
   docs: { source: { type: "code" } },
 }
 
+export const WithSections: ComponentStory<typeof Select> = () => (
+  <Select
+    id="select-grouped"
+    label="label"
+    items={groupedMockItems}
```

Comment:
Is it possible to show the `groupedMockItems` in this code snippet? I'd prefer we have an example so consumers know how to use this feature 😄 

<img width="313" alt="image" src="https://user-images.githubusercontent.com/820635/214440859-5eaa7c2c-8dca-48a8-8281-0126d09c0d47.png">